jupyter_server_proxy/handlers.py

```diff
@@ -650,6 +654,9 @@ async def ensure_process(self):
                 proc = SupervisedProcess(self.name, *cmd, env=server_env, ready_func=self._http_ready_func, ready_timeout=timeout, log=self.log)
                 self.state['proc'] = proc
 
+                if not cmd:
+                    return
+
```

Comment:
Is this called to late? Should `self.state['proc']` be set to a new `SupervisedProcess` object? The idea was that if the command is unset, it shouldn't be supervised - right?

If it is correct, a comment is needed to clarify why it is correct in my mind as this is quite unexpected for me.

Reply:
Besides this, the changes looks good to me!

Reply:
Like it is now, it won't be awaited readiness either. I'll push a commit to move this logic upwards.